_tutorials/blog_feed.md

```diff
@@ -0,0 +1,60 @@
+---
+title: User blog
+position: 3
+right_code: |
+    Skeleton
+    ``` html
+    <html>
+    <head><title>User blog</title></head>
+    <body onload="fetchBlog()">
+    <div class="container">
+      <h2>Welcome to my blog!</h2>
+      <div class="list-group" id="postList"></div>
+    </body>
+    </html>
+    ```
+    Script Line
```

Comment:
NOTE: While this way of including js is good for a _very_ beginner user,
We will want to encourage people to use modern Javascript best practices. That includes npm, es6, require & import statements. No changes are needed now, but we'll have a later story to update this tutorial to a simplest version of best practice.

Reply:
@relativityboy I think for the "first" basic tutorial vanilla javascript could be used. But I completely agree, we need to cover different javascript frameworks